bedrock/mozorg/templates/mozorg/about/leadership.html

```diff
@@ -845,6 +845,29 @@ <h3 class="group-title" id="board-corp">{{ _('Mozilla Corporation') }}</h3>
           <p>He attended Stanford University, where he earned MBA, MA, and BA degrees, and did graduate studies in Computer Science.</p>
         </div>
       </article>
+
+      <article id="hugh-molotsi" data-id="hugh-molotsi" class="vcard has-bio" itemscope itemtype="http://schema.org/Person">
+        <figure class="headshot">
+          <img class="photo" src="{{ static('img/mozorg/about/leadership/hugh-molotsi.jpg') }}" alt="" itemprop="image" width="160" height="160">
+          <figcaption><h4 class="fn" itemprop="name">Hugh Molotsi</h4></figcaption>
+        </figure>
+
+        <div class="person-info">
+          <ul class="utility">
+            <li><a class="link" href="{{ url('mozorg.about.leadership') }}#hugh-molotsi">{{ _('Link to this bio') }}</a></li>
+          </ul>
+        </div>
+
+        <div class="person-bio">
+          <p>Hugh Molotsi is the founder and CEO of Ujama, a messaging platform for communities that makes it easy for people to help each other. Hugh is a seasoned product and technology leader and an accomplished innovator who has helped launch several disruptive products.</p>
+
+          <p>Hugh had a 22-year career at Intuit where his last position was Engineering Fellow and Vice President of Innovation. Hugh worked on QuickBooks and other small business offerings. Hugh led the launch of several new businesses, including QuickBooks Enterprise and Intuit Payments, which deliver over $1B in annual  revenue today. Hugh was an inaugural member of Intuit’s Diversity Council, leading to the formation of a dozen employee networks, including Intuit’s faith-based networks. Hugh has been a dedicated supporter of early career development and helped develop Intuit’s Rotational Development Program.</p>
+
+          <p>High is the co-author of “The Intrapreneur’s Journey”, a book on how to develop a culture of innovation at large companies. Hugh holds a Master of Science degree in Computer Engineering, and a Bachelor of Science degree in Computer Engineering Technology. Hugh is also an American Leadership Forum senior fellow and a certified leadership development trainer.</p>
```

Comment:
Should this be Hugh?

```suggestion
          <p>Hugh is the co-author of “The Intrapreneur’s Journey”, a book on how to develop a culture of innovation at large companies. Hugh holds a Master of Science degree in Computer Engineering, and a Bachelor of Science degree in Computer Engineering Technology. Hugh is also an American Leadership Forum senior fellow and a certified leadership development trainer.</p>
```